### PR TITLE
feat: remove '/preview' and '/beta'

### PR DESCRIPTION
**Pull request by Victoremepunto**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,10 +6,10 @@
 VERSION ?= 0.0.1
 
 # CHANNELS define the bundle channels used in the bundle.
-# Add a new line here if you would like to change its default config. (E.g CHANNELS = "preview,fast,stable")
+# Add a new line here if you would like to change its default config. (E.g CHANNELS = "fast,stable")
```
Comment (psav) [inline Makefile:9]: Not necessary to change this file, this is for OLM I believe for the bundles there.

File: controllers/reconcile.go [hunk 2]
```diff
@@ -684,12 +678,10 @@ func (r *FrontendReconciliation) getFrontendPaths() []string {
 	frontendPaths := r.Frontend.Spec.Frontend.Paths
 	defaultPath := fmt.Sprintf("/apps/%s", r.Frontend.Name)
 	defaultBetaPath := fmt.Sprintf("/beta/apps/%s", r.Frontend.Name)
```
Comment (psav) [inline controllers/reconcile.go:680]: Beta path can go too?

File: controllers/reconcile.go [hunk 3]
```diff
@@ -684,12 +678,10 @@ func (r *FrontendReconciliation) getFrontendPaths() []string {
...
-	defaultPreviewPath := fmt.Sprintf("/preview/apps/%s", r.Frontend.Name)
 
 	if r.Frontend.Spec.AssetsPrefix != "" {
 		defaultPath = fmt.Sprintf("/%s/%s", r.Frontend.Spec.AssetsPrefix, r.Frontend.Name)
 		defaultBetaPath = fmt.Sprintf("/beta/%s/%s", r.Frontend.Spec.AssetsPrefix, r.Frontend.Name)
```
Comment (psav) [inline controllers/reconcile.go:684]: Beta path can go too?

File: controllers/reconcile.go [hunk 4]
```diff
@@ -700,10 +692,6 @@ func (r *FrontendReconciliation) getFrontendPaths() []string {
 		frontendPaths = append(frontendPaths, defaultBetaPath)
```
Comment (psav) [inline controllers/reconcile.go:692]: Beta path can go too?

File: tests/e2e/basic-frontend/02-assert.yaml [hunk 5]
```diff
@@ -33,6 +33,4 @@ spec:
               protocol: TCP
           volumeMounts:
             - name: config
-              mountPath: /opt/app-root/src/build/stable/operator-generated
-            - name: config
-              mountPath: /opt/app-root/src/build/preview/operator-generated
+              mountPath: /opt/app-root/src/build/stable/operator-generated
```
Comment (psav) [inline tests/e2e/basic-frontend/02-assert.yaml:36]: new line?

**Review by psav — CHANGES_REQUESTED** [review]
A few questions and a change or two :)

File: Makefile (hunk 1 above)
Comment (Victoremepunto) [reply Makefile]: thanks, I rolled this change back

File: controllers/reconcile.go (hunk 2 above)
Comment (Victoremepunto) [reply controllers/reconcile.go]: Thought about it, I added the removal as a separate commit now, thanks!

File: controllers/reconcile.go (hunk 3 above)
Comment (Victoremepunto) [reply controllers/reconcile.go]: :+1:  done!

File: tests/e2e/basic-frontend/02-assert.yaml (hunk 5 above)
Comment (Victoremepunto) [reply tests/e2e/basic-frontend/02-assert.yaml:36]: no need, file is properly formatted. I'd love to pass through lint and automatic format processes to be honest.

File: controllers/reconcile.go (hunk 4 above)
Comment (Victoremepunto) [reply controllers/reconcile.go]: gone

**Review by psav — APPROVED** [review]

**Review by adamrdrew — APPROVED** [review]
I have never loved a PR more than this one

**Pull request merged**